src/VCR/MatchObserver.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace VCR;
+
+class MatchObserver
```

Comment:
At the moment the MatchObserver is a singleton. It could also be a value object which gets filled with data and then thrown away.